src/Raven.Client/ServerWide/CompactSettings.cs

```diff
@@ -9,14 +9,17 @@ public class CompactSettings
         public bool Documents { get; set; }
 
         public string[] Indexes { get; set; }
+        
+        public bool SkipOptimize { get; set; }
```

Comment:
We need to be more specific and explicitly say that it's about indexes so maybe `SkipOptimizeIndexes`.

This option should be exposed in the studio CC @ml054